src/lib/authentication/pages/Login/LoginWithLocalAccount/LoginWithLocalAccount.js

```diff
@@ -91,11 +91,12 @@ class LoginWithLocalAccount extends Component {
       const errors = getIn(error, 'response.data.errors', []);
 
       if (_isEmpty(errors)) {
-        const message = getIn(error, 'response.data.message', null);
-        if (message) {
-          actions.setSubmitting(false);
-          actions.setErrors({ message });
+        let message = getIn(error, 'response.data.message', null);
```

Comment:
If we return 'Unknown error' as default in the [getIn](https://lodash.com/docs/4.17.15#get) we could avoid the `if`
`const message = getIn(error, 'response.data.message', 'Unknown error');`

Reply:
If the response is defined then we should look at `error.response.data.message` (the message returned by the server).
Otherwise there was a network issue and we can obtain the message from `error.message` (this message is provided by the javascript client, usually "Network Error").
But you're right that I can nest them in a one-liner like so:
```javascript
const message = getIn(error, 'response.data.message', getIn(error, 'message', 'Unknown error'));
```